vpc.tf

```diff
@@ -1,45 +1,52 @@
-data "aws_availability_zones" "available" {}
-
-provider "aws" {
-  profile = "reut"
-	region = var.aws_region
-}
 
 # VPC
 resource "aws_vpc" "reut_vpc" {
   cidr_block = var.vpc_cidr
+
+  tags = {
+    Name = "reut_vpc"
+  }
+}
+
+#Nat gatways for the private subnets
+resource "aws_nat_gateway" "private_subnet" {
+  count         = length(var.subnets_cidr_private)
+  allocation_id = aws_eip.nat.*.id[count.index]
+  subnet_id     = aws_subnet.private_subnet.*.id[count.index]
   tags = {
-  Name = "reutvpc"    
+    Name = "NAT_${count.index}"
   }
-}  
+}
 
 # Internet Gateway
-resource aws_internet_gateway "reut_igw1" {
-vpc_id = aws_vpc.reut_vpc.id
-tags = {
-Name = "main_igw"
-    }
+resource aws_internet_gateway "reut_igw" {
+  vpc_id = aws_vpc.reut_vpc.id
+  tags = {
+    Name = "igw"
+  }
```

Comment:
why `reut_igw1` and not just `reut_igw` ?

Reply:
כי היה לי אחד כזה כבר